### DISCUSSION TITLE
Open feeds in external Browser

### DISCUSSION THREAD

**Discussion by MasothU** [tughi/aggregator-android#92] [Ideas]
Hello,
First of all thank you very much for working on this new version. I am really looking forward to working with this version.
One request. Can you add the option to open the feeds in an external browser instead of a built in?

**Comment by tughi** [comment]
See #75 for solution.

**Reply by MasothU** [reply]
Hello,
I think this is still not yet implemented? At least in preview 34?
Your solution is regarding opening the rss by clicking on their header /icon.
In this question/request I asked for the possibility to open the feeds in the normal browser directly and not the built-in browser functionality.
I want to open them as normal tabs, say in Firefox for example.
